test/components/document_list_test.rb

```diff
@@ -0,0 +1,117 @@
+require 'component_test_helper'
+
+class DocumentListTest < ComponentTestCase
+  def component_name
+    "document-list"
+  end
+
+  test "fails to render a document list when no parameters are given" do
+    assert_empty render_component({})
+  end
+
+  test "fails to render a document list when an empty array is passed in" do
+    assert_empty render_component(items: [])
+  end
+
+  test "renders a document list correctly" do
+    render_component(
+      items: [
+        {
+          link: {
+            text: "School behaviour and attendance: parental responsibility measures",
+            path: "/government/publications/parental-responsibility-measures-for-behaviour-and-attendance"
+          },
+          metadata: {
+            public_updated_at: Time.zone.parse("2017-01-05 14:50:33 +0000"),
+            document_type: "statutory_guidance"
+          }
+        },
+        {
+          link: {
+            text: "School exclusion",
+            path: "/government/publications/school-exclusion"
+          },
+          metadata: {
+            public_updated_at: Time.zone.parse("2017-07-19 15:01:48 +0000"),
+            document_type: "statutory_guidance"
+          }
+        }
+      ]
+    )
+    li = ".app-c-document-list__item-title"
+
+    assert_select "#{li} a[href='/government/publications/parental-responsibility-measures-for-behaviour-and-attendance']", text: "School behaviour and attendance: parental responsibility measures"
+    assert_select ".app-c-document-list__attribute time[datetime='2017-01-05T14:50:33+00:00']", text: "5 January 2017"
+    assert_select ".app-c-document-list__attribute", text: "Statutory guidance"
+
+    assert_select "#{li} a[href='/government/publications/school-exclusion']", text: "School exclusion"
+    assert_select ".app-c-document-list__attribute time[datetime='2017-07-19T16:01:48+01:00']", text: "19 July 2017"
+
+    #check date is rendered correctly when it is GMT
```

Comment:
Not sure we need these timezone tests given the component simply handles what it’s given